service/worker/indexer/indexer.go

```diff
@@ -58,13 +58,14 @@ type (
 	}
 	// Indexer used to consumer data from kafka then send to ElasticSearch
 	Indexer struct {
```

Comment:
it's cleaner to separate MigrationIndexer from Indexer to avoid further confusion